robot.py

```diff
@@ -137,11 +132,33 @@ def controllerInit(self, config):
             rta = ctrlConfig['RIGHT_TRIGGER_AXIS']
             ctrls.append(Controller(ctrl, dz, lta, rta))
         return ctrls
+    
+    def initElastic(self, auton_config):
+        
+        self.log("initElastic ran")
 
-    def initSwervometer(self, config):
-        self.log("initSwervometer ran")
+        #used to check for difference in selection on elastic
+        self.ELASTIC_BOT_COLOR = "GREEN"
+        self.ELASTIC_BOT_POSITION = "Z"
+        self.ELASTIC_BOT_AUTON = "Z_ZERO_NOTE"
+
+        autonPlans = list(filter(lambda k: "NOTE" in k, auton_config.keys()))
+        elastic = Elastic(autonPlans, self.savedSwervometerConfig["ALL_ACTIVE_POSITIONS"])
+
+        elastic.autonDisplay() # "Display" are sendableChoosers
 
-        if (config['TEAM_IS_RED']):
+        if self.savedSwervometerConfig['TEAM_IS_RED']:
```

Comment:
As above, I think this could become `if self.config["SWERVOMETER"]["TEAM_IS_RED"]`